doc/ubuntu_release_process.md

```diff
@@ -115,46 +117,50 @@ Last, we need to push our tag above.
-
-    ## just tag current so you have a revert point.
-    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
-    $ git checkout ubuntu/xenial
-    $ cherry-pick dc2bd79949
-
-The tool will:
-
-  * add a new file to debian/patches/ named cpick-<commit>-topic that is dep-8 compatible
-  * add that patch to debian/patches/series
-  * verify that the patch applies (quilt push -a)
-  * give you a chance to edit formatted the debian/changelog entry
-  * prompt you to commit the change.
+```
+  $ git fetch upstream; git checkout upstream/ubuntu/xenial -b ubuntu/xenial
+  $ cherry-pick <new_cherry_pick_commitish>
+  $ cherry-pick <commit_hash_2>
+  # Put up two PRs for review ubuntu/$release and ubuntu/daily/$release
+  $ git push <your_remote> ubuntu/$release
+  $ git push <your_remote> ubuntu/daily/$release
+```
+
+
+The `cherry-pick` tool will:
+
+  * add a new file to debian/patches/ named cpick-<commit>-topic that is
+    dep-3 compatible
+  * append file to debian/patches/series
+  * calls quilt push -a
+  * prompts for commiting the debian/changelog
+  * checkout upstream/ubuntu/daily/$release branch
+  * merge ubuntu/$release into ubuntu/daily/$release
+  * git revert the local cherry-pick from ubuntu/daily/$release so daily builds
+    continue to work
 
 From here you follow along with the snapshot upload process from
 [`dch --release` and beyond](#upstream-snapshot-process).
 
-**Note**: After doing uploading, in order to keep the daily builds working, we will then have to revert change.  This is because the recipe will try to build from trunk, and will fail to apply your cherry-picked patch.  This makes sense... it will grab trunk and then try to apply patches, but the cherry-picked patch will already exist.
-
-To do this:
-
-    $ git checkout ubuntu/xenial  # or whatever release.
-    $ new-upstream-snapshot -v --update-patches-only
-    $ git push upstream HEAD
 
-That will produce 1 or 2 commits on the branch with summary like:
+#### Fixing daily builds after cherry-pick
 
-  * refresh patches against master commit 2d6e4219
-  * drop cherry picks included in master commit 2d6e4219
+**Note**: After uploading a cherry-pick release, the daily builds will break
+unless we revert the cherry-picks from the ubuntu/daily/$release branch. This
+is because the recipe will try to merge from tip of master, and will get
+conflicts re-applying your cherry-picked patch because it already exists.
+The `cherry-pick` tool takes care of reverting any cherry picks from
+ubuntu/daily/$release branches.
 
-After doing that you can go to the recipe pages (see below) and click
-build-now.
+After ubuntu/daily/$release PRs are merged, go to the recipe pages (see below)
+and click build-now.
 
 ### Adding a quilt patch to debian/patches ###
 This is generally needed when we are disabling backported feature from tip
-in order to retain existing behavior on an older series.
+in order to retain existing behavior on a stable release.
```

Comment:
I'd leave it as is, we do this for non-lts older series which is still in support; older captures both stable and non-stable (non-lts is what you mean).